src/Features/CSharpTest/ConvertToRecord/ConvertToRecordCodeRefactoringTests.cs

```diff
@@ -4596,8 +4583,18 @@ private class CodeFixTest :
         {
             public CodeFixTest()
             {
+                ReferenceAssemblies = ReferenceAssemblies.Net.Net60;
```

Comment:
Why did this need to change?

Reply:
➡️ `SolutionTransforms` was doing two things:

1. Setting reference assemblies
2. Enabling nullable reference types